cpp/ql/lib/experimental/cryptography/CryptoArtifact.qll

```diff
@@ -0,0 +1,316 @@
+import cpp
+private import experimental.cryptography.CryptoAlgorithmNames
+import semmle.code.cpp.ir.dataflow.TaintTracking
+
+/*
+ * A cryptographic artifact is a DataFlow::Node associated with some
+ * operation, algorithm, or any other aspect of cryptography.
+ */
+
+abstract class CryptographicArtifact extends Expr { }
+
+// /**
```

Comment:
Any reason why this is commented out? Can you please include a comment explaining why, and why it is important to leave in, or else delete it.